### PR TITLE
iOS Keychain issue: removing entitlements for iOS

### PR DESCRIPTION
**Pull request by heyzooi**
removing entitlements for iOS solves the issue of saving items in the keychain

### CODE REVIEW

File: app/App_Resources/iOS/build.xcconfig [hunk 1]
```diff
@@ -5,3 +5,4 @@
 // DEVELOPMENT_TEAM = YOUR_TEAM_ID;
 ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 ASSETCATALOG_COMPILER_LAUNCHIMAGE_NAME = LaunchImage;
+CODE_SIGN_ENTITLEMENTS = ;
```
Comment (thomasconner) [inline app/App_Resources/iOS/build.xcconfig:8]: What does this do?

Comment (heyzooi) [reply app/App_Resources/iOS/build.xcconfig:8]: it overwrites the default values after you create a new project in NativeScript

Comment (thomasconner) [reply app/App_Resources/iOS/build.xcconfig:8]: Is this required in order to interact with the iOS keychain on the iOS simulator?

Comment (heyzooi) [reply app/App_Resources/iOS/build.xcconfig:8]: exactly

**Comment by tejasranade** [comment]
@thomasconner is this good to go? If so, please merge.

**Pull request merged**